config_defaults/subtests/docker_cli/negativeusage.ini

```diff
@@ -70,24 +70,6 @@ subarg = -e,PATH=/tmp,%%(FQIN)s,true
 stderr = exec\: \"true\"\: executable file not found in \$PATH
 extcmd = 127
 
-[docker_cli/negativeusage/iv5]
-subcmd = run
-subarg = --tmpfs,/dev,%%(FQIN)s
-stderr =  no such file or directory
-extcmd = 127
-
-[docker_cli/negativeusage/iv6]
-subcmd = run
-subarg = --tmpfs,:rw,%%(FQIN)s
-stderr =  open /proc/self/task/1/attr/exec: no such file or directory
-extcmd = 125
-
-[docker_cli/negativeusage/iv7]
-subcmd = run
-subarg = --tmpfs,.,%%(FQIN)s
-stderr =  Message: Failed to open /dev/null - open /dev/null: permission denied
-extcmd = 125
```

Comment:
Gah, *squeek*, *squeek* ahh ha!  It wasn't a speck on my glasses!  Really should clean them more often :grinning: 

```Timestamp: 2016-10-27 19:10:20.065693229 +0000 UTC
Code: System error

Message: Failed to open /dev/null - open /dev/null: permission denied

Frames:
---
0: setupRootfs
Package: github.com/opencontainers/runc/libcontainer
File: rootfs_linux.go@81
---
1: Init
Package: github.com/opencontainers/runc/libcontainer.(*linuxStandardInit)
File: standard_init_linux.go@59
---
2: StartInitialization
Package: github.com/opencontainers/runc/libcontainer.(*LinuxFactory)
File: factory_linux.go@247
---
3: initializer
Package: github.com/docker/docker/daemon/execdriver/native
File: init.go@35
---
4: Init
Package: github.com/docker/docker/pkg/reexec
File: reexec.go@26
---
5: main
Package: main
File: docker.go@20
---
6: main
Package: runtime
File: proc.go@188
---
7: goexit
Package: runtime
File: asm_amd64.s@1998
docker: Error response from daemon: Cannot start container a2fdec625689f1646c4da64a1f647d2b2065ea31ea4cf86a9abb376337491a6a: [9] System error: invalid character 'r' looking for beginning of value.
```

Right, I think Afom showed that to me before.  Yuk.  I half-wonder if we shouldn't file a BZ on that.  I'd much prefer if docker just threw a simple error like: "tmpfs paths must be absolute".  Then if so, that begs the question of, shouldn't that format of output be caught by ``OutputGood``?

Sorry, lots of pondering, few answers.